src/app/state/accountHistorySlice.ts

```diff
@@ -26,11 +26,13 @@ export interface AccountHistoryState {
 const initialState: AccountHistoryState = {
   trades: [],
   orderHistory: [],
+  // these tables are component specific and should not be in app state
```

Comment:
Makes sense. But to ensure I get the idea right: the underlying data that is needed to derive the tables (all user trades) should be in global user state, right?

Reply:
Yes. The general data should be stored globally so it is available to many components. The formatting of the data for the specific use in the component must be done locally.
It means the app can keep the general data updated in one place (state) and these changes can be fed to the various components automatically (by using slices).